novaideo/graphql/mutations.py

```diff
@@ -43,6 +52,7 @@ class Input:
         title = graphene.String()
         text = graphene.String()
         keywords = graphene.List(graphene.String)
+        attached_files = graphene.List(Upload)
```

Comment:
please add a comment that we currently get a List of null, that the Upload object deserialization doesn't currently work.